src/components/buttons/button-accessibility-image-alt.jsx

```diff
@@ -0,0 +1,163 @@
+import ButtonIcon from './button-icon.jsx';
+import ButtonProps from '../base/button-props.js';
+import ButtonStateClasses from '../base/button-state-classes.js';
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+/**
+ * The ButtonAccessibilityImageAlt class provides functionality for changing text color in a document.
+ *
+ * @class ButtonAccessibilityImageAlt
+ * @uses ButtonProps
+ * @uses ButtonStateClasses
+ *
+ */
+class ButtonAccessibilityImageAlt extends React.Component {
+    static key = 'AccessibilityImageAlt';
+
+    static propTypes = {
+        /**
+         * The editor instance where the component is being used.
+         *
+         * @instance
+         * @memberof ButtonAccessibilityImageAlt
+         * @property {Object} editor
+         */
+        editor: PropTypes.object.isRequired
+    };
+
+
+    constructor(prop, context) {
+        super(prop, context);
+
+        const image = this.props.editor.get('nativeEditor').getSelection().getSelectedElement();
+
+        this.state = {
+            element: image,
+            imageAlt: image ? image.getAttribute('alt') : ''
+        };
+    }
+
+    /**
+     * Lifecycle. Renders the UI of the button.
+     *
+     * @method render
+     * @return {Object} The content which should be rendered.
+     */
+    render() {
+        var cssClass = 'ae-button ' + this.getStateClasses();
```

Comment:
@wincent 
- I can replace `var` with `let`/`const`, I just didn't do it to avoid to many changes

- Concerning `getStateClasses()` and HOC's in general (in this project), we might want to review them and use them only when it makes sense. I might be wrong but from what I've seen some of them could just be utility functions. (Let me know if I understood that correctly)